app/src/main/java/com/bielczy/mvvmclean/core/app/roomDataBase/EpisodesDao.kt

```diff
@@ -0,0 +1,18 @@
+package com.bielczy.mvvmclean.core.app.roomDataBase
+
+import androidx.lifecycle.LiveData
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+
+interface EpisodesDao {
```

Comment:
W zasadzie trochę wyprzedziłeś kolejne lekcje, bo będziemy implementować takie Dao, tylko metody nieco się będą różnić. Zostawiam do Twojej decyzji czy teraz je tutaj usunąć i napisać na nowo w jednym z następnych modułów, czy jeśli dojdziesz do tej lekcji z implementacją Dao to po prostu sobie te metody podmienisz :)